core-bundle/src/Resources/contao/forms/FormCaptcha.php

```diff
@@ -64,8 +64,6 @@ public function __construct($arrAttributes=null)
 
 		$this->arrAttributes['maxlength'] = 2;
 		$this->strCaptchaKey = 'c' . md5(uniqid(mt_rand(), true));
-		$this->arrAttributes['required'] = true;
-		$this->arrConfiguration['mandatory'] = true;
```

Comment:
In case you are wondering: I had to remove this to prevent an `An invalid form control with name='c150bc…' is not focusable.` error when the form is sent within the first 5 seconds and the input field does not have a value yet.